service-commands/scripts/hosts.js

```diff
@@ -82,7 +82,7 @@ if (cmd === 'add') {
 } else if (cmd === 'add-remote-host') {
   let lines = readFileIntoArray()
   if (!verifyNotAlreadyAdded(lines)) {
-    console.log('Local dev mapping already exists!')
+    console.log('Local dev mapping already exists! If you want to update your hosts file, please run the `remove` command and then re-run `add-remote-host`.')
```

Comment:
isn't this the add-remote-host condition? message seems redundant

Reply:
this is to clear up confusion around running `add` before `remove`. @vicky-g was trying to do that and getting this error and didn't realize that you need to first run `remove` and then `add`, so this is just to clear that up

the `add` vs `add-remote-host` is unrelated